src/System.Security.Cryptography.Encoding/tests/Asn1/Reader/ReadOctetString.cs

```diff
@@ -207,7 +207,7 @@ public static void TryCopyOctetStringBytes_Fails(PublicEncodingRules ruleSet, st
             Assert.Equal(expectedHex, output.AsSpan(0, bytesWritten).ByteArrayToHex());
         }
 
-        private static void TryCopyOctetStringBytes_Throws(
+        private static void TryCopyOctetStringBytes_ThrowsCore(
```

Comment:
Why?

Reply:
I think that was needed because corefxlab had a different version of xunit (the project failed during the Discovering phase, IIRC).

I don't think the tests are going to be as sharable.  Ideally this change wouldn't be here, but it's not major.  (Partly this is "any test that *changes* means there's more product risk since the reader changed; any *new* tests are less scary")

Reply:
I'm prefectly fine with reverting it if you don't see a reason to keep it. The Xunit in CoreFX was updated because it failed to discover many tests which were in turn not run at all, so I would definitely expect the issue to be solved by the update.